core/app_controller.py

```
@@ -0,0 +1,649 @@
+            if not content_service:
+                self._logger.error("Content service not available")
+                return None
+            
+            # Create content DTO with basic information
+            from data.models import PlatformType
+            platform_type = None
+            if platform:
+                try:
+                    platform_type = PlatformType(platform.lower())
+                except ValueError:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError)`

(SIM105)